cmd/vic-init/ops.go

```diff
@@ -69,6 +70,9 @@ func (t *operations) HandleSessionExit(config *tether.ExecutorConfig, session *t
 			return
 		}
 
+		// incredibly basic throttle
```

Comment:
Please tag this as a TODO. This indicates that we have issues around synchronization; we should not be putting in random sleeps.